src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt

```diff
@@ -18,14 +18,17 @@ class DossierContextFactory(
 ) {
   fun createContext(recallId: RecallId): DossierContext {
     val recall = recallRepository.getByRecallId(recallId)
-    val currentPrisonName = prisonLookupService.getPrisonName(recall.currentPrison!!)
-    return DossierContext(recall, currentPrisonName)
+    val currentPrisonId = recall.currentPrison!!
+    val currentPrisonName = prisonLookupService.getPrisonName(currentPrisonId)
+    val currentPrisonIsWelsh = prisonLookupService.isWelsh(currentPrisonId)
```

Comment:
Feels a bit clunky? Would it be better to have a single `prisonLookupService.getPrison` which returns ID, name, active & isWelsh?

Reply:
I was thinking to change (what is now) the `Api.Prison` class to add an `isWelsh` property and decorating the response object from the Prison register - but currently we only need to know this in one place so I went with putting this only in the LookupService.  I'm fine with looking at the alternative.